src/pineko/scale_variations.py

```diff
@@ -0,0 +1,266 @@
+"""Module to generate scale variations."""
+import pathlib
+from typing import Tuple
+
+import numpy as np
+import pineappl
+import rich
+from eko import beta
+
+from . import check
+
+AS_NORM = 1.0 / (4.0 * np.pi)
+OrderTuple = Tuple[int, int, int, int]
+"""Tuple representing a PineAPPL order."""
+
+
+def sort_qcd_orders(order: OrderTuple) -> int:
+    """Define the sorting for an order. In particular the order is sorted according to the as order."""
+    return order[0]
+
+
+def ren_sv_coeffs(m, max_as, logpart, which_part, nf):
```

Comment:
I would try to improve the names.

- `nf` is fine as it is
- `m`, `max_as`, `logpart`, `which_part` are all orders (or order-tuple elements), but from their names they seem very different from each other: I'd try to find more explicit and harmonized names

I will try to propose some names myself.